triedb/pathdb/history_indexer.go

```diff
@@ -126,11 +121,11 @@ func deleteIndexMetadata(db ethdb.KeyValueWriter, typ historyType) {
 // of historical data (e.g., state or trie node changes) atomically.
 type batchIndexer struct {
 	index   map[stateIdent][]uint64 // List of history IDs for tracked state entry
-	pending int                     // Number of entries processed in the current batch.
-	delete  bool                    // Operation mode: true for unindex, false for index.
-	lastID  uint64                  // ID of the most recently processed history.
-	typ     historyType             // Type of history being processed (e.g., state or trienode).
-	db      ethdb.KeyValueStore     // Key-value database used to store or delete index data.
+	pending int                     // Number of entries processed in the current batch
```

Comment:
I'd like to remove this kind of changes, unless it breaks the linter. The thing is, if I want to understand how a line came to be in the codebase, and I do a git blame, it will lead me to this commit and not the one that originally created the line.